.gitignore

```diff
@@ -26,3 +26,6 @@ CMakeLists.txt.user.*
 vtest/html
 thirdparty/mupdf-qt
 
+/BuildRelease.bat
+/BuildClean.bat
+/BuildInstall.bat
```

Comment:
after these changes these Build*.bat files should not be part of the commit, should they?